satellite/satellitedb/overlaycache.go

```diff
@@ -1395,61 +1389,62 @@ func (cache *overlaycache) UpdateCheckIn(ctx context.Context, node overlay.NodeC
 			(
 				id, address, last_net, protocol, type,
 				email, wallet, free_bandwidth, free_disk,
-				audit_success_count, total_audit_count, uptime_success_count, total_uptime_count,
-				created_at, updated_at, last_contact_success, last_contact_failure,
-				audit_reputation_alpha, audit_reputation_beta, uptime_reputation_alpha, uptime_reputation_beta,
-				contained, disqualified,
-				piece_count, major, minor, patch, hash, timestamp, release, latency_90
+				uptime_success_count, total_uptime_count, 
+				last_contact_success,
+				last_contact_failure,
+				audit_reputation_alpha, audit_reputation_beta, uptime_reputation_alpha, uptime_reputation_beta
 			)
 			VALUES (
 				$1, $2, $3, $4, $5,
 				$6, $7, $8, $9,
-				0, 0, $10, 1,
-				current_timestamp, current_timestamp, $11, $12,
-				$21, $22, $13, $14,
-				false, null,
-				0, 0, 0, 0, '', $15, false, 0
+				$10::bool::int, 1,
+				CASE WHEN $10 IS TRUE THEN current_timestamp
+					ELSE '0001-01-01 00:00:00+00'
+				END,
+				CASE WHEN $10 IS FALSE THEN current_timestamp
+					ELSE '0001-01-01 00:00:00+00'
+				END,
+				$11, $12, $13, $14
 			)
 			ON CONFLICT (id)
 			DO UPDATE
 			SET
-				total_uptime_count=nodes.total_uptime_count+1,
-				uptime_reputation_alpha=$18::numeric*nodes.uptime_reputation_alpha + $19::numeric*(1+$20)/2,
-				uptime_reputation_beta=$18::numeric*nodes.uptime_reputation_beta + $19::numeric*(1-$20)/2,
+				address=$2,
+				last_net=$3,
```

Comment:
Good catch. I hadn't noticed that `last_net` wasn't being updated. How about `protocol`; can that ever change?

Reply:
yeah i wasn't sure about protocol. let me look into adding it